plugins/forms/plugin.js

```diff
@@ -277,3 +277,24 @@ CKEDITOR.plugins.add( 'forms', {
 		}
 	}
 } );
+
+/**
+ * Namespace containing helper functions for Forms Plugin.
+ *
+ * @since 4.10.2
+ * @private
+ * @singleton
+ * @class CKEDITOR.plugins.forms
+ */
+CKEDITOR.plugins.forms = {
+	/**
+	* Sets dialog required value.
```

Comment:
Maybe add some more info, e.g. it's based on algorithm described in specification.